examples/plot_example_workflow.py

```diff
@@ -0,0 +1,151 @@
+"""
+===========================================================
+Example ``autoreject`` preprocessing workflow including ICA
+===========================================================
+
+This example demonstrates how to visualize data when preprocessing
+with :mod:`autoreject` and discusses decisions about when and which
+other preprocessing steps to use in combination.
+"""
+
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#         Mainak Jas <mjas@mgh.harvard.edu>
+#         Apoorva Karekal <apoorvak@uoregon.edu>
+#
+# License: BSD (3-clause)
+
+# sphinx_gallery_thumbnail_number = 2
+
+# %%
+# First, we download resting-state EEG data from a Parkinson's patient
+# from OpenNeuro. We will do this using ``openneuro-py`` which can be
+# installed with the command ``pip install openneuro-py``.
+
+import os
+import os.path as op
+import matplotlib.pyplot as plt
+import openneuro
+
+import mne
+import autoreject
+
+dataset = 'ds002778'  # The id code on OpenNeuro for this example dataset
+subject_id = 'pd14'
+
+target_dir = os.path.join(
+    os.path.dirname(autoreject.__file__), '..', 'examples', dataset)
```

Comment:
I'm not sure this is a good place to store the data. Wouldn't that end up in my `site-packages` directory? That's not one anyone would want, I suppose?

Reply:
Maybe a `TempDir`?

Reply:
I think I'd go for

```python
mne.get_config()[ 'MNE_DATA']
```